admin/onboarding/onboard.md

```diff
@@ -16,13 +25,46 @@ Updated 6/3/21
       - [Backend](#backend)
     - [Specs](#specs)
 
+
 ## Understanding iNeedA\<br>
 iNeedA\<br> (Group) is founded on the belief that everyone is an ally to everyone else. We value phycological safety and promote an environment where people are able to present ideas and expect to be given constructive criticism. We value individuals who are open about their struggles, are not shy about asking for help, and are constantly improving through constructive criticism. 
 
+## Understanding Eucalyptus
+Eucalyptus is a light-weight online bullet journal tool that allows users to perform rapid logging, prioritization, and more. Eucalyuptus supports three types of bullets: `note`, `task`, and `event`. Eucalyptus also supports the inclusion of additional entries in the form of plaintext documents or images. 
+
+Eucalyptus uses IndexDB as a database for storing and retrieving bullets. Our API supports storing bullets and retriving bullets based on queries (Date, Bullet Type, etc.)
+
+## Building Eucalyptus
```

Comment:
What does build mean in this context? Our code doesn't need any builds (local or repo)?

Reply:
The code gets auto-tested and such, but we don't use any major packages that need compiling or things like that. Deployment is it's own separate thing

Reply:
that's why they generate "automatically" and don't "require extra actions from the developer"